package/MDAnalysis/analysis/hbonds/wbridge_analysis.py

```diff
@@ -0,0 +1,813 @@
+      of a water bridge existence, or see :attr:`~WaterBridgeAnalysis.table`.
+
+   .. attribute:: table
+
+      A normalised table of the data in
+      :attr:`WaterBridgeAnalysis.timeseries`, generated by
+      :meth:`WaterBridgeAnalysis.generate_table`. It is a
+      :class:`numpy.recarray` with the following columns:
+
+          0. "time"
+          1. "donor_index"
+          2. "acceptor_index"
+          3. "donor_resnm"
+          4. "donor_resid"
+          5. "donor_atom"
+          6. "acceptor_resnm"
+          7. "acceptor_resid"
+          8. "acceptor_atom"
+          9. "distance"
+          10. "angle"
+
+      It takes up more space than :attr:`~WaterBridgeAnalysis.timeseries` but
+      it is easier to analyze and to import into databases or dataframes.
+
+
+      .. rubric:: Example
+
+      For example, to create a `pandas.DataFrame`_ from ``h.table``::
+
+         import pandas as pd
+         df = pd.DataFrame.from_records(w.table)
+
+
+      .. versionchanged:: 0.17.0
+         The 1-based donor and acceptor indices (*donor_idx* and
+         *acceptor_idx*) are deprecated in favor of 0-based indices.
+
+"""
+import numpy as np
+import logging
+
+from .hbond_analysis import HydrogenBondAnalysis
+from MDAnalysis.lib.NeighborSearch import AtomNeighborSearch
+from MDAnalysis.lib.log import ProgressMeter, _set_verbose
+
+logger = logging.getLogger('MDAnalysis.analysis.wbridges')
+
+class WaterBridgeAnalysis(HydrogenBondAnalysis):
+    """Perform a water bridge analysis
+
+    The analysis of the trajectory is performed with the
+    :meth:`WaterBridgeAnalysis.run` method. The result is stored in
+    :attr:`WaterBridgeAnalysis.timeseries`. See
+    :meth:`~WaterBridgeAnalysis.run` for the format.
+
+    The default atom names are taken from the CHARMM 27 force field files, which
+    will also work for e.g. OPLS/AA in Gromacs, and GLYCAM06.
+
+    *Donors* (associated hydrogens are deduced from topology)
+      *CHARMM 27*
+        N of the main chain, water OH2/OW, ARG NE/NH1/NH2, ASN ND2, HIS ND1/NE2,
+        SER OG, TYR OH, CYS SG, THR OG1, GLN NE2, LYS NZ, TRP NE1
+      *GLYCAM06*
+        N,NT,N3,OH,OW
+
+    *Acceptors*
+      *CHARMM 27*
+        O of the main chain, water OH2/OW, ASN OD1, ASP OD1/OD2, CYH SG, GLN OE1,
+        GLU OE1/OE2, HIS ND1/NE2, MET SD, SER OG, THR OG1, TYR OH
+      *GLYCAM06*
+        N,NT,O,O2,OH,OS,OW,OY,P,S,SM
+      *amber99sb-ildn(Gromacs)*
+        OC1, OC2 of the main chain
+
+    See Also
+    --------
+    :ref:`Default atom names for water bridge analysis`
+
+
+    .. versionchanged:: 0.7.6
+       DEFAULT_DONORS/ACCEPTORS is now embedded in a dict to switch between
+       default values for different force fields.
+    """
+    def __init__(self, universe, selection1='protein', selection2='not resname SOL', water_selection = 'resname SOL',
+                 selection1_type='both', update_selection1=True, update_selection2=True, update_water_selection = True,
```

Comment:
pep8: no space around `=` in argument list

Reply:
should the default for `update_selection[12]` be `False` because they are likely static?

Reply:
@orbeckst Thank you for the suggestion. The default value and the relevant doc has been rewritten.